.github/files/metadata.schema

```diff
@@ -0,0 +1,14 @@
+{
+    "type" : "object",
+    "properties" : {
+        "name" : {"type" : "string"},
+        "display-name" : {"type" : "string"},
+        "summary" : {"type" : "string", "maxLength": 50},
+        "description" : {"type" : "string"},
+        "docs" : {"type" : "string"},
+        "maintainers" : {"type" : "array", "items": {"type": "string", "enum": ["launchpad.net/~canonical-is-devops"]}},
+        "issues" : {"type" : "string"},
+        "tags" : {"type" : "array", "items": {"type": "string"}}
```

Comment:
Would be possible to check for "assumes" too?
```yaml
description: Description
assumes:
  - k8s-api

Reply:
The workflows also support machine charms, so I don't think it's a good idea